UKHomeOffice/cerberus-service
# Cop 9675 handling tasklist cards

## Description
This PR looks at the ability of enabling the handling of different RoRo Modes, with particular focus at the different variations of RoRo Tourist (COP-9675). COP-9674 is the implementation of the findings of COP-9675.

**JIRA Ticket**: https://support.cop.homeoffice.gov.uk/browse/COP-9675

## To Test
- Pull & run against dev.
> RoRo Tourist With Vehicle:
- The Following columns should be displayed on Task List Page: **Driver** | **VRN** | **Booking** | **Co-Travellers**

> RoRo Tourist Foot Passenger (Single Passenger) & RoRo Tourist Foot Passengers (Group Passengers):
- The Following columns should be displayed on Task List Page: **Primary Traveller** | **Document** | **Booking** | **Co-Travellers**


## Developer Checklist

\* Required
